palestirnative/palestirnative
# fix: pagination number and text

## Description

This PR fixes #8 and makes the text of previous and next buttons translated.

### How to test it

Test the following scenario for both `/boycott` and `/alternative`
1. Open the list page
2. Navigate to the bottom
3. Click on next
4. Navigate to the bottom again
5. **The current page number should be 2 now**
6. Change the language
7. **The buttons text should change**

### Screenshots
You should have something like that
![image](https://github.com/palestirnative/palestirnative/assets/10698248/f778a945-0cb9-42c7-9adc-d4a864386c78)
